server.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/acm-uiuc/arbor/server"
+	"github.com/acm-uiuc/groot-api-gateway/config"
```

Comment:
so arbor is now a package you import. it contains 4 parts, the proxy, security, server and service and basically manage everything about the gateway. groot-api-gateway is the name of the api gateway app. and it just contains the routes and settings we want.